package.json

```diff
@@ -300,31 +300,31 @@
   },
   "dependencies": {
     "archiver": "^2.1.1",
-    "cs-jsforce-metadata-tools": "^1.2.10",
```

Comment:
Is there a reason to remove this?  There are methods used in the extension from this fork.  If necessary to have recent updates from the upstream jsforce-metadata-tools, please make the repository public and update from the upstream master,.